docker/node/Dockerfile

```
@@ -0,0 +1,18 @@
+# base image
+FROM node:10
```

we should use the same node version than the one located in `package.json engines` key 
=> so if you have to use a different version of node, please update the value of `engines` accordingly


besides, could it be possible to use `alpine` here, instead of the default `stretch` distro?

```suggestion
FROM node:10-alpine
```

and if you think it's also possible to use the latest LTS version of node (not required but could be nice):

```suggestion
FROM node:12-alpine
```